src/Util/Reflection.php

```diff
@@ -37,6 +37,8 @@ public function getProperty($methodName)
         if (preg_match('/^('.$pattern.')(.+)$/i', $methodName, $matches)) {
             return $matches[2];
         }
+        
+        return null;
```

Comment:
it doesn't change the outcome but show that's the desired returning a `null` value, unless your consider the phpdoc to be enough here?